pytorch_lightning/accelerators/tpu.py

```diff
@@ -25,6 +26,15 @@
 class TPUAccelerator(Accelerator):
     """Accelerator for TPU devices."""
 
+    def __init__(self) -> None:
+        """
+        Raises:
+            MisconfigurationException:
+                If the TPU device is not available.
+        """
+        if not _XLA_AVAILABLE:
```

Comment:
should we add another one for TPU_AVAILABLE?
https://github.com/PyTorchLightning/pytorch-lightning/issues/3104